cardano-shell.cabal

```diff
@@ -45,30 +45,28 @@ library
     , specs
   build-depends:
       aeson
-    , base >=4.7 && <5
+    , base >=4.11 && <5
     , binary
     , bytestring
-    , cardano-prelude
     , Cabal
+    , cardano-prelude
```

Comment:
I don't like the fact that people are reordering dependencies and removing comments from them.